### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by ojstilo** [GoldpawsStuff/BigGameMenu#1]
Hi! I love this addon, but I have a few suggestions to make it even better:

- **microinteractions** - add mouseover effects and onclick effects to make it feel more functional and alive and to provide interaction feedback for users
- **background** - most of the times it doesn't feel like a main game menu, like something which is "on top of the interface" - consider adding background. My suggestion is a solid dark color with some opacity - it could be even something as simple as full black with 30% opacity - subtly covering whole game window/screen
- **fix frame strata** - my Bartender action bars appear "above" the game menu. Please, fix the strata level so the menu is always on top.

**Comment by goldpaw** [comment]
I planned the first two, just never got around to them. Then somebody wanted the addon and I just released it anyway. So. Adding them back to the todo-list now! :)

The buttons are just WoWs own textures, so it was fast to make. I'll add mouseover effects and stuff in one of the upcoming days! 

As for the backdrop, I have some plans, but there is nothing in WoW that looks good and highres enough, quite frankly, so I'll have to make that myself. Might take "a few more days" to squeeze that job in! :)

As for the frame strata, the menu is parented to the regular game menu and inherits its strata. Does your bartender bars appear above the default menu too, if they have been that far up I mean? What strata are your bars set to? And what wow version, btw? BCC, Classic Era or Retail?

**Comment by ojstilo** [comment]
It is regarding the Retail WoW.

I have just tested the Bartender with default menu and it does not appear above it.

AFAIK I have never changed Bartender's strata, so it must be at the default.

Usually I don't have the bars this close to the middle, but I moved one for the Mage Tower and that's when I noticed this issue. I also thought though that this might be a bigger issue once you'll add some background.

I attach screenshots with both default menu and BigGameMenu with Bartender4. (Had to move it a little bit more to the middle for default)

![WoWScrnShot_121421_161405](https://user-images.githubusercontent.com/46416839/146027511-2c4e3450-15ba-44da-98dd-c0cf648d67c9.jpg)
![WoWScrnShot_121421_161247](https://user-images.githubusercontent.com/46416839/146027516-8cc70e7e-6b91-4821-88ec-3da6e38f39e2.jpg)
